client/gutenberg/extensions/giphy/index.js

```diff
@@ -0,0 +1,23 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { settings as giphySettings } from './settings.js';
+import edit from './edit';
+import './editor.scss';
+import './style.scss';
+
+export const { name } = giphySettings;
+
+export const settings = {
```

Comment:
This seems pretty unusual. Shall we export `name` and `settings` directly here as we do in other blocks?

Reply:
To confirm, are you asking why this block has a separate `settings.js` file at all as opposed to including all settings right in `index.js`?

Reply:
Yep! Most other Jetpack blocks have an index which exports `name` and `settings`, which are defined there:

https://github.com/Automattic/wp-calypso/blob/8a2f6265eb9381a1fabdbe7b01d8253bae3dc506/client/gutenberg/extensions/tiled-gallery/index.js#L91
https://github.com/Automattic/wp-calypso/blob/8a2f6265eb9381a1fabdbe7b01d8253bae3dc506/client/gutenberg/extensions/tiled-gallery/index.js#L102

https://github.com/Automattic/wp-calypso/blob/8a2f6265eb9381a1fabdbe7b01d8253bae3dc506/client/gutenberg/extensions/markdown/index.js#L17-L19

Is the indirection and re-export of parts of the object exported from settings here providing value, or can we define the settings right here? I see that the `apiKey` is shared, but that might be better to move into a shared constants file.